src/api/cloud/project/serving/namespaces/cloud-project-io-stream.v6.service.js

```diff
@@ -0,0 +1,6 @@
+angular
+  .module('ovh-api-services')
+  .service('OvhApiCloudProjectServingNamespaceV6', ($resource) => $resource('/cloud/project/:serviceName/ai/serving/:namespaceId', {
```

Comment:
When creating a route we should try to follow the API route . 
Here the path is `/cloud/project/:serviceName/ai/serving/:namespaceId` so ideally your service should be named `OvhApiCloudProjectAiServing`
`namespaceId` is a parameter so you would use a `.get()` for `/cloud/project/:serviceName/ai/serving/:namespaceId`  but with a `.query()` the path would be `/cloud/project/:serviceName/ai/serving`(no `Namespace` in the path)

Reply:
reworked here 8b8c1fd9b2505b77313fb3e7b00f0336c73513ad